docs/quickstart/ccd.md

```diff
@@ -26,15 +28,10 @@ Once you're on a self hosted subscription, you can proceed with this quickstart
 
 ## Install the following tools
 
-* [`strm`][cli], the STRM Privacy cli (optional). You need this to control your STRM resources, and to simulate some events.
-  You could do without, and use the [STRM console][console].
-* `make` (optional): The Helm repository contains handy `make` targets such as _install_, _upgrade_, _wipe_. None of
-  these are mandatory, one can also execute the commands in the [`Makefile`][makefile] by hand.
-* [`kubectl`](https://kubernetes.io/docs/tasks/tools/)
-* [`helm`](http://helm.sh) (required): This Kubernetes package manager is used for installing (and upgrading) your STRM
-  customer data plane.
-* [`telepresence`][telepresence] (optional): This tool allows your development computer to become _part of_ the k8s
-  cluster, so that you can directly access k8s services.
+* [`strm`][cli], the STRM Privacy cli. You need this to control your STRM resources, and to simulate some events.
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/), the Kubernetes cli.
+* [`helm`](http://helm.sh): This Kubernetes package manager is used for installing (and upgrading) your STRM
```

Comment:
The other list items have commas instead of a colon. I have no preference what to use.

Reply:
Thinking about it, I guess I have a slight preference for colons.